data/json/monsters/turrets.json

```diff
@@ -20,7 +20,7 @@
     "armor_cut": 16,
     "vision_night": 3,
     "revert_to_itype": "bot_turret",
-    "starting_ammo": { "9mm": 30 },
+    "starting_ammo": { "9mm": 1500 },
```

Comment:
This should stay 30, I think; this is a turret improvised by player which uses a normal MP5 magazine, which is 30 rounds.

Reply:
Ah yes didn't realize it was an improvised turret. Which turret is the one that appears in science stations and fires 9mm?

Reply:
There is none.

Reply:
Really? The 9mm 100 round science lab turrets are gone? Were they ever replaced with anything similar or is it just zeds now?

Reply:
They were obsoleted and replaced with CROWS.
They were then un-obsoleted to make these survivor 9mm turrets.